commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/UnicycleDrive.scala

```diff
@@ -65,11 +94,11 @@ trait UnicycleDrive extends Drive {
   import UnicycleControllers._
 
   object unicycleTasks {
-    class ContinuousDrive(forward: Signal[Dimensionless], turn: Signal[Dimensionless])
-                         (implicit drive: Drivetrain) extends ContinuousTask {
+    class ContinuousClosedDrive(forward: Signal[Dimensionless], turn: Signal[Dimensionless])
+                               (implicit drive: Drivetrain) extends ContinuousTask {
       override def onStart(): Unit = {
         val combined = forward.zip(turn).map(t => UnicycleSignal(t._1, t._2))
-        drive.setController(toClosedDrive(combined))
+        drive.setController(parentClosedLoop(combined))
```

Comment:
That is the method to pass drive signals to to run closed loop at the parent (whoever's extending `UnicycleDrive`) level. So here we would be running left-right control at the `TwoSidedDrive` level.